static/js/segmentForm.js

```diff
@@ -2,33 +2,13 @@
 /* eslint-disable */
 
 !function(){
-  /**
```

Comment:
Note that the changes here and the removed files above reflect the change in how we load segment. We now allow segment to load the libs as configured on their site.